### ISSUE TITLE
🌟 [EPIC] Accessibility

### ISSUE THREAD

**Issue by Tirokk** [Human-Connection-Community/Human-Connection-New-20201003#3633]
> [<img alt="sushidave" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/sushidave) **Authored by [sushidave](https://github.com/sushidave)**
_<time datetime="2020-06-12T22:06:41Z" title="Saturday, June 13th 2020, 12:06:41 am +02:00">Jun 13, 2020</time>_

---

## :rocket: Feature

Make the network accessible according to the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG21/).

### User Problem

Handicapped people e.g. with physical or cognitive disabilities cannot use the network to its full extent. This issue solves the problem.

### Implementation

**Help wanted:**

- People who add the guidelines to the [documentation](https://docs.human-connection.org/human-connection/) (for improvement and future development).
- People who know about the WCAG guidelines, assess the existing network and identify rooms for improvement.
- People who propose solutions.
- People who code it.
- People who test it.

**Steps:**

1. Documentation:
- [ ] Include accessibility in the [documentation](https://docs.human-connection.org/human-connection/)

2. Requirements:
- [ ] Identify what needs to be improved
- [ ] Propose solutions

3. Code implememtation:
- [ ] Registration, login, logout
- [ ] Profile page
- [ ] Account settings
- [ ] Newsfeed
- [ ] Search
- [ ] Filters
- [ ] Single post
- [ ] Post editor #3625 
- [ ] Comments
- [ ] Notifications
- [ ] Moderation
- [ ] Administration
- [ ] any others?

### Design & Layout

To be assessed, see point 2 above.

### Validation

Adhere to the WCAG standards. 

Benefits due to accessibility and inclusion: 
- More users able to use the network. 
- Less support issues. 
- Higher user satisfaction. 
- Promote accessibility as a USP feature of this network.

### Additional context

As discussed with @Raimundo-dev yesterday.
